src/pages/nfts/[address].tsx

```diff
@@ -564,6 +568,23 @@ const NftShow: NextPage<NftPageProps> = ({ marketplace }) => {
           {marketplace.name}
         </title>
         <link rel="icon" href={marketplace.logoUrl} />
+        <meta property="og:type" content="website" />
+        {hostname && (<meta property="og:site_name" content={hostname} />)}
+        {data?.nft ? (
+          <>
+            <meta property="og:title" content={data?.nft.name + ' | '+ marketplace.name} />
+            <meta property="og:image" content={data?.nft.image} />
+            <meta property="og:description" content={data?.nft.description} />
+          </>
+        ):
```

Comment:
The nft is already being queried from the server. Pick off the name, image, and description so you have it on the first run of the render. It wont add anymore load time since all the fields are already on base nft record.

Reply:
@kespinola Done!